samples/tests/playwright.config.ts

```diff
@@ -15,8 +16,21 @@ const config: PlaywrightTestConfig = {
       name: 'chromium',
       use: {
         browserName: 'chromium',
-        viewport: { width: 1280, height: 720 }
-      }
+        launchOptions: {
+          args: [
+            '--font-render-hinting=none', // Ensures that fonts are rendered consistently.
```

Comment:
we really need common place for all these things :(

Reply:
I am thinking of move composite tests and these tests into a common folder (after GA), then maybe it will be the best time to extract common stuffs together :D